app/views/includes/footer-categories.njk

```diff
@@ -39,4 +39,19 @@
       </li>
     </ul>
   </div>
+  <div class="govuk-footer__section">
```

Comment:
It’s a little crazy but we use this https://github.com/alphagov/pay-selfservice/blob/master/app/utils/display_converter.js is where permissions are injected into the template.

Reply:
Perfect thanks, I'll check against that to render the third section selectively.

Reply:
Addressed in https://github.com/alphagov/pay-selfservice/pull/1089/commits/1a4f01f06a7cf9c09ac839b19cccdac0b94fc204